### PR TITLE
networking: set MTU on a VLAN subinterface for the bond using ignition config works

### PR DESCRIPTION
**Pull request by HuijingHei**
Set MTU on a VLAN subinterface for the bond using ignition config and check
- verify MTU on the bond matches config
- verify MTU on the VLAN subinterface for the bond matches config
- verify ip address on the VLAN subinterface for the bond matches config

The ignition config is generated using nm-initrd-generator according to
https://docs.fedoraproject.org/en-US/fedora-coreos/sysconfig-network-configuration/
$ kargs="bond=bond0:eth1,eth2:mode=active-backup,miimon=100:9000 \
       ip=10.10.10.10::10.10.10.1:255.255.255.0:staticvlanbond:bond0.100:none:9000: \
       vlan=bond0.100:bond0"
$ /usr/libexec/nm-initrd-generator -s -- $kargs

### CODE REVIEW

**Comment by HuijingHei** [comment]
This is the following PR for https://github.com/coreos/fedora-coreos-config/pull/1401

File: tests/kola/networking/mtu-on-bond-ignition/config.bu [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    - path: /etc/NetworkManager/system-connections/bond0.100.nmconnection
+      mode: 0600
+      contents:
+        inline: |
+          [connection]
+          id=bond0.100
+          type=vlan
+          interface-name=bond0.100
+          [ethernet]
+          mac-address-blacklist=
```
Comment (ravanelli) [inline tests/kola/networking/mtu-on-bond-ignition/config.bu]: I know you used the doc to create it. A curiosity, make sense to have these items even with empty values?

Comment (HuijingHei) [reply tests/kola/networking/mtu-on-bond-ignition/config.bu]: Thanks @ravanelli , maybe no need to keep them, I will remove all items with empty values

Comment (dustymabe) [reply tests/kola/networking/mtu-on-bond-ignition/config.bu]: ehh - I would use exactly what was generated by `nm-initrd-generator`. This makes it easy to regenerate in the future.

**Review by dustymabe — APPROVED** [review]
LGTM - though I would keep the NM config files exactly as they were generated by `nm-initrd-generator`.

File: tests/kola/networking/mtu-on-bond-ignition/config.bu (hunk 1 above)
Comment (HuijingHei) [reply tests/kola/networking/mtu-on-bond-ignition/config.bu]: Does it need to remove line like `uuid=d1171f53-2502-45dd-9199-698a778bf3bb`?

Comment (dustymabe) [reply tests/kola/networking/mtu-on-bond-ignition/config.bu]: no - we don't need to remove the uuid lines I don't think.

Comment (HuijingHei) [reply tests/kola/networking/mtu-on-bond-ignition/config.bu]: Just to confirm, this means we should remove the uuid lines?

Comment (dustymabe) [reply tests/kola/networking/mtu-on-bond-ignition/config.bu]: No. I would think here that we don't need to remove **any** lines from the output of `nm-initrd-generator`.

Comment (HuijingHei) [reply tests/kola/networking/mtu-on-bond-ignition/config.bu]: OK, thanks for your confirmation

**Pull request merged**